tests/CoreFX/CoreFX.issues.json

```diff
@@ -216,6 +216,36 @@
             ]
         }
     },
+    {
+        "name": "System.Runtime.InteropServices.Tests",
+        "enabled": true,
+        "exclusions": {
+            "namespaces": null,
+            "classes": null,
+            "methods": [
+                {
+                    "name": "System.Runtime.InteropServices.MarshalTests.SetComObjectData",
+                    "reason": "Xunit.Sdk.ThrowsException Assert.Throws() Failure\\nExpected: typeof(System.PlatformNotSupportedException)\\nActual:   typeof(System.ArgumentNullException): Value cannot be null."
```

Comment:
Nit: I typically just put reason as `outdated` for tests that are outdated. The actual exception is not interesting for them.